### ISSUE TITLE
Facebook ads extension error

### ISSUE THREAD

**Issue by Bankina** [facebookincubator/Facebook-For-OpenCart#109]
Hello. I installed it, followed the instructions and this error keeps showing " There is an error with Facebook Ads Extension setup. Click on Facebook Ads Extension, Manage Settings, go to Advanced options and click on Delete Settings to restart the setup.
Please contact Facebook via our Github if this error keeps showing."
I did that. Nothing changed.

**Comment by Bankina** [comment]
P.S. OC 3.0.2.0

**Comment by shue249** [comment]
@Bankina Thank you for your feedback. Few questions.

1. How many products do you have on your OpenCart store?

2. What is your webstore URL?

**Comment by Bankina** [comment]
Hello,

18544 in the moment.

https://abarauto.bg/

**Comment by shue249** [comment]
@Bankina The reason for the failure could be due to out of memory when fetching all the products for the initial product sync. Are you selling all the products on your store? Will you be able to disable products which are no selling anymore? 
https://github.com/facebookincubator/Facebook-For-OpenCart/issues/72

**Comment by Bankina** [comment]
@shue249 Yes. I sell them all. I have even more, so I can't reduce them. Any solution?

**Comment by shue249** [comment]
@Bankina Are you using MYSQL database? If so, can you try to increase the wait_timeout and max_allowed_packet variables of your MYSQL server.

You can refer to the section "Syncing of OpenCart products to Facebook catalog" in our FAQ - https://github.com/facebookincubator/Facebook-For-OpenCart/blob/master/FAQ.md

**Comment by supermatjar** [comment]
I have the same problem
my store is 
supermatjar.com

only 8000 products !!

**Comment by shue249** [comment]
@supermatjar Can you pls help us to create a new ticket

**Comment by supermatjar** [comment]
Yes but how to create

في الجمعة، ٢٢ مارس، ٢٠١٩ ٣:٥٦ ص shue249 <notifications@github.com> كتب:
